server/eb/app.py

```diff
@@ -24,49 +24,53 @@
     from server.app.app import Server
     from server.data_common.matrix_loader import MatrixDataCacheManager
 except Exception:
-    logging.exception("Exception importing server modules")
+    logging.critical("Exception importing server modules", exc_info=True)
     sys.exit(1)
 
 try:
     dataroot = os.getenv("CXG_DATAROOT")
-    if dataroot is None:
-        logging.error("CXG_DATAROOT environment variable must be set")
-        sys.exit(1)
+    app_config = AppConfig()
+
+    config_file = "config.yaml"
+    if os.path.exists(config_file):
```

Comment:
future enhancement:  the config file should be pulled from a environment variable, with a hard-wired backup.  And it should allow for an S3 URL (eg, store it in the dataroot) - I would just open it with the DataLocator class.    I'll file this as a ticket so as to not extend this PR.

Reply:
I like the idea of having the option to store the config in s3.  Good idea.

Reply:
filed as #1273